src/Inputs/ComboBox.tsx

```diff
@@ -0,0 +1,241 @@
+import React, {
+    useState,
+    useEffect,
+    useCallback,
+    Children,
+    isValidElement,
+    SyntheticEvent, 
+} from 'react';
+import styled, { withTheme, DefaultTheme } from 'styled-components';
+import {
+    flex,
+    scroll,
+    position,
+    transition,
+    clickable,
+    darken,
+} from '@Utils/Mixins';
+import { useTransition } from '@Utils/Hooks';
+import { LabelLayout, LabelLayoutProps,InputFragment } from '@Layouts';
+
+const NUM_VISIBLE_SELECTIONS = 3;
+
+var options:any = [];
+
+var isMounted=false;
+
+const SPEED = 'normal';
+
+let myWindow = window as any;
+
+export interface SelectProps extends LabelLayoutProps {
+    disabled?: boolean;
+    placeholder?: string;
+    value?: string | number;
+    theme: DefaultTheme;
+    onChange?: Function;
+    limit?: number;
+    numVisible:number;
+}
+
+const _Select: React.FC<SelectProps> = ({

```

Comment:
Why does combobox copy code from select and not use select?

Reply:
its mostly select, I'm going to delete the unnecessary props that were useless in the combobox

Reply:
No but why are we copying the select component code and not using the select component directly